.travis.yml

```diff
@@ -44,15 +44,15 @@ script:
   - echo "Working copy disk usage"; du -chs .
   - grunt --version
   - grunt help
-  - grunt --quiet --timer
```

Comment:
Both the `--quiet` option and `GDT_QUIET` env var are supposed to only affect desktop notifications (i.e. growl). If the option affects the error reporting from Grunt, we should change the option to avoid conflicts.